### DISCUSSION TITLE
有没可能弄个7B的呢

### DISCUSSION THREAD

**Discussion by gaangon** [ymcui/Chinese-Mixtral#19] [General]
感谢，可否弄个7b的，我感觉对于普通人来讲，7b和8x7b的区别没那么大，7b的4Q量化版本估计个人电脑还能跑起来（8g显存）

**Comment by ymcui** [comment]
Mixtral原版只有8x7B的。如需单模型7B的，你可以用我们的Llama-2系列的模型：https://github.com/ymcui/Chinese-LLaMA-Alpaca-2

**Reply by rexchujie** [reply]
大神，那个Mistral 7B的模型，会考虑训练个中文支持好的吗？

**Comment by ILG2021** [comment]
我也很期待，Mistral 7b instruct v0.2 刚发布，能力超llama2 13b
